ViAn/GUI/mainwindow.ui

```diff
@@ -17,35 +17,44 @@
    <double>1.000000000000000</double>
   </property>
   <widget class="QWidget" name="centralWidget">
-   <layout class="QVBoxLayout" name="verticalLayout_2">
+   <layout class="QVBoxLayout" name="verticalLayout">
```

Comment:
Becouse they are not used for anything outside the .ui file there is no need for a descriptive name